README.md

```diff
@@ -457,6 +462,17 @@ Other Style Guides
   ];
   ```
 
+  <a name="arrays--no-unnecessary-arrays"></a>
+  - [4.7](#arrays--no-unnecessary-arrays) Avoid create unnecessary arrays because of a performance reason.
+
+  ```javascript
+  // bad
+  const bar = [...foo].map(mapper);
+
+  // good
+  const bar = Array.prototype.map.call(foo, mapper);
```

Comment:
best is `Array.from(foo, mapper)` - `Array.prototype.map.call` isn't equivalent, since that only accepts arrays and arraylikes - spread and `Array.from` also accepts iterables.